apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/AppService.java

```diff
@@ -153,6 +161,24 @@ public App createAppInLocal(App app) {
     return createdApp;
   }
 
+  @Transactional
+  public App createAppAndAddRolePermission(
+      App app, Set<String> admins
+  ) {
+    // the aop of @Transactional fail when invoke by internal class's method
+    App createdApp = this.createAppInLocal(app);
```

Comment:
createAppInLocal has `@Transactional` too.

It's ok if we also add `createAppAndAddRolePermission` on method createAppAndAddRolePermission?

Reply:
Looks like we could change createAppInLocal's modifier to private and remove the `@Transactional`?

Reply:
Done